### ISSUE TITLE
[Idea] Entity Black/Whitelist

### ISSUE THREAD

**Issue by MrDj200** [MrDj200/Attractor#4]
Make an Entity Black/Whitelist as mentioned in #2

**Comment by mrdeadlocked** [comment]
This would be great!

**Comment by MrDj200** [comment]
yeah, it would. But I'm currently just maintaining the mod to remove bugs etc.
But I will start active developement again. 
Too many games to play atm...
